test/Microsoft.VisualStudio.Sdk.TestFramework.Tests/Mocks/GreetMock.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using static System.Reflection.BindingFlags;
+
+public class GreetMock : IGreet, IMockServiceWithClientCallback
+{
+    public event EventHandler<string>? GreetingSent;
+
+    public event EventHandler? OperationComplete;
+
+    public object ClientCallback { get; set; } = new object();
```

Comment:
nit: I don't think we should initialize this with an object that doesn't implement the expected interface.
But I suppose you're trying to avoid declaring the property as nullable, which makes sense since we guarantee that the base class will initialize it. So you could do this instead:

```suggestion
    public object ClientCallback { get; set; } = null!;
```